### PR TITLE
Windows Firewall State Discovery and Posh Disable Windows Firewall TTPs

### PR DESCRIPTION
**Pull request by w0rk3r**
Proposing a modification in the current Windows Defender Firewall Procedure by bfuzzy and adding a Posh command to it.

### CODE REVIEW

File: defense-evasion/70a57910-a56a-4b60-bcb4-84b7a56076ab.yml [hunk 1]
```diff
@@ -1,18 +1,22 @@
...
+      command: |
+        netsh advfirewall set allprofiles state off
+    psh:
+      command: |
+        Set-NetFirewallProfile -All -Enabled False
 metadata:
   authors:
     - bfuzzy1
+    - w0rk3r
   tags: []
```
Comment (khyberspache) [inline defense-evasion/70a57910-a56a-4b60-bcb4-84b7a56076ab.yml:21]: seeing as this is changing a system configuration, lets add the `destructive` tag so that users have to explicitly enable this TTP before using it.

**Review by khyberspache — CHANGES_REQUESTED** [review]
Looks solid - just want to add a destructive tag to the disabling TTP

File: defense-evasion/70a57910-a56a-4b60-bcb4-84b7a56076ab.yml (hunk 1 above)
Comment (w0rk3r) [reply defense-evasion/70a57910-a56a-4b60-bcb4-84b7a56076ab.yml]: Added

**Comment by w0rk3r** [comment]
Done

**Comment by khyberspache** [comment]
Thanks! Great contributions.

**Pull request merged**